.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+  - /bin/true
```

Comment:
Why do we have this noop here and all the actual build stuff under `before_deploy` below? Surely building is useful for continuous integration even when we're not deploying?